erpnext/controllers/queries.py

```diff
@@ -468,26 +468,20 @@ def warehouse_query(doctype, txt, searchfield, start, page_len, filters):
 	conditions, bin_conditions = [], []
 	filter_dict = get_doctype_wise_filters(filters)
 
-	sub_query = """ select round(`tabBin`.actual_qty, 2) from `tabBin`
-		where `tabBin`.warehouse = `tabWarehouse`.name
-		{bin_conditions} """.format(
-		bin_conditions=get_filters_cond(doctype, filter_dict.get("Bin"),
-			bin_conditions, ignore_permissions=True))
-
-	query = """select `tabWarehouse`.name,
-		CONCAT_WS(" : ", "Actual Qty", ifnull( ({sub_query}), 0) ) as actual_qty
-		from `tabWarehouse`
+	query = """select tw.name,
+		CONCAT_WS(" : ", "Actual Qty", ifnull(round(tb.actual_qty, 2), 0 )) actual_qty
+		from `tabWarehouse` tw left join `tabBin` tb
+		on tb.warehouse = tw.name {bin_conditions}
```

Comment:
instead of tb, tw use proper table name `tabBin` and `tabWarehouse` so that you don't need to use replace method in line 482 and 484

Reply:
Okay, fixed that.